resources/web/docs_js/components/modal.js

```diff
@@ -0,0 +1,18 @@
+import {h} from "../../../../../node_modules/preact";
+import {connect} from "../../../../../node_modules/preact-redux";
+import {closeModal} from "../actions/modal";
+
+export const Modal = props => {
+  return props.isOpen && (
+    <div id="settings_modal_bg">
+      <div id="settings_modal">
+        <div class="settings_modal-close" onClick={props.closeModal}>&times;</div>
```

Comment:
`class` vs `className`

Reply:
totally unintentionally forgot to change this, but I just learned that preact supports `class`. I will update it to className just to be consistent with regular React just in case.